packages/create-vite/index.js

```diff
@@ -311,6 +314,9 @@ function isValidPackageName(projectName) {
  * @param {string} projectName
  */
 function toValidPackageName(projectName) {
+  if (projectName === '.') {
+    projectName = path.basename(path.resolve())
```

Comment:
[`defaultProjectName` is initialized to `'vite-project'`](https://github.com/vitejs/vite/blob/407c0043039b7ba51baf9fbacd1f46fac838d340/packages/create-vite/index.js#L138) when the target dir is unspecified. Wouldn't it be easier to change `'vite-project'` to `path.basename(path.resolve())` instead? I think that would be more user-friendly as the user wouldn't need to enter anything.

Reply:
That's more consistent with initializing in other directories and  it's more user-friendly! I have implemented that. 😃